VALUES.md

```diff
@@ -0,0 +1,27 @@
+# Core Values
+
+## Our Mission
+
+At its core, our project is a collection of people working towards a shared goal.
+Improving people's lives. This can be done through making software faster,
+it can be done by making empathetic APIs. This can be done by making a reliable platform,
```

Comment:
The comma at the end of this line is a comma splice. Make it a period that ends the sentence and the next clause is a sentence to itself, or put the word `and` after the comma.

Reply:
fixed